imapclient/client.go

```diff
@@ -1062,3 +1093,8 @@ type loginCommand struct {
 type logoutCommand struct {
 	cmd
 }
+
+// PtrT Get a pointer to any type
+func PtrT[T any](arg T) *T {
```

Comment:
- I don't want to expose this function in the library's API.
- I don't think the use of generics is warranted here, since the function is only called with one type.
- Just using `&` instead of calling this function works fine.